Randomizer.py

```diff
@@ -0,0 +1,24 @@
+import random
+
+
+def randomly(list_of_students):
+    student_dict_reverted = {}
+    for elem in list_of_students:
+        student_dict_reverted[elem] = 0
+    print('Do you want to make your choice?', '\n', 'Press y to do it, n to exit')
+    while True:
+        choice2 = input()

```

Comment:
Why `choice2`, not `choice1` or `choice9000`

Reply:
Remained from the old code, already fixed[2]